### ISSUE TITLE
Space between initials

### ISSUE THREAD

**Issue by DerWeh** [nschloe/betterbib#212]
In case of multiple first names, `betterbib` puts no space, see e.g.

```shell
$ betterbib doi-to-bibtex 10.1103/physrevb.4.2412
%comment{This file was created with betterbib v4.0.1.}


@article{blackman1971,
 author = {Blackman, J.A. and Esterling, D.M. and Berk, N.F.},
 doi = {10.1103/physrevb.4.2412},
 number = {8},
 pages = {2412-2428},
 source = {Crossref},
 url = {http://dx.doi.org/10.1103/physrevb.4.2412},
 volume = {4},
 journal = {Phys. Rev. B},
 publisher = {American Physical Society (APS)},
 title = {Generalized Locator---Coherent-Potential Approach to Binary Alloys},
 issn = {0556-2805},
 year = {1971},
 month = oct,
}
```

`bibtex`, however, needs a space, that is, `author = {Blackman, J. A. and Esterling, D. M. and Berk, N. F.},`.

**Comment by nschloe** [comment]
Reference?

**Comment by nschloe** [comment]
> The problem is that bibtex understands Blackman, J.A. as a person with first name J.A. rendering the name as J. Blackman, not a person with two first names:

I cannot reproduce this. MWE:

```latex
\documentclass{article}
\usepackage{biblatex}

\begin{filecontents}{\jobname.bib}
@article{einstein,
  author = {Einstein, A.X.},
  title = {Some title},
  journaltitle = {Annalen der Physik}
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}
  \nocite{einstein}
  \printbibliography
\end{document}
```
Output:

![screenshot](https://user-images.githubusercontent.com/181628/136190505-a1f81f39-1d6c-45f6-bb6a-f395b2e0d0a5.png)

**Comment by DerWeh** [comment]
That's because you choose to spell out the full first name, use `giveninits=True` to see the difference.
See:
```text
\documentclass{article}
\usepackage[giveninits=true]{biblatex}

\begin{filecontents}{\jobname.bib}
@article{einstein,
  author = {Einstein, A.X.},
  title = {Some title},
  journaltitle = {Annalen der Physik}
}
@article{fulleinstein,
  author = {Einstein, Albert Xylophon},
  title = {Some title},
  journaltitle = {Annalen der Physik}
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}
  \nocite{einstein,fulleinstein}
  \printbibliography
\end{document}
```

![Screenshot from 2021-10-06 13-50-11](https://user-images.githubusercontent.com/22542812/136196997-25bb4cf9-7cf3-4291-8728-4507d4a34b84.png)

Whereas printing the full name, as you did:

```tex
\documentclass{article}
\usepackage[giveninits=false]{biblatex}

\begin{filecontents}{\jobname.bib}
@article{einstein,
  author = {Einstein, A.X.},
  title = {Some title},
  journaltitle = {Annalen der Physik}
}
@article{fulleinstein,
  author = {Einstein, Albert Xylophon},
  title = {Some title},
  journaltitle = {Annalen der Physik}
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}
  \nocite{einstein,fulleinstein}
  \printbibliography
\end{document}
```
![Screenshot from 2021-10-06 13-52-06](https://user-images.githubusercontent.com/22542812/136197158-63d8322d-4500-4f93-9787-64d94e334b9d.png)

----
In principle, the bibtex file should optimally contain all information available, and the bibtex style or the biblatex configuration decides how it should be presented, e.g. abbreviating names are giving using et al. after a certain number of names.
This is especially important, if you submit the manuscript to Journals, which use their on personal configurations.

**Comment by nschloe** [comment]
> That's because you choose to spell out the full first name

I don't understand. I only give the initials, how do I "spell out the full first name"?

**Comment by nschloe** [comment]
Ah, you mean that by not providing `giveninits`, I tell biblatex to use to the "full" first name, and it understands "A.X." as full first name.

I think I understand the problem now. Given that [this blog entry](https://clauswilke.com/blog/2015/10/02/bibtex/) from 2015 recommends the same,

> Therefore, always abbreviate names in your Bibtex file, and leave a space between initials, [...]

I'll see about changing this in betterbib.

**Comment by nschloe** [comment]
Fix released in 4.0.2. Thanks for the concise report!
